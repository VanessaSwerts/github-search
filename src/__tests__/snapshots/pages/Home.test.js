import React from 'react'
import { render } from '@testing-library/react'
import { Home } from '../../../pages'

describe('Testes de Snapshot da Home', () => {

  test('Renderização da tela Home', () => {
    const { asFragment } = render(<Home />)

    expect(asFragment(<Home />)).toMatchSnapshot()
  })

})