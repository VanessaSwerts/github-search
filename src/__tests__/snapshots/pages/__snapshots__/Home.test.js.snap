// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Home Renderização da tela Home 1`] = `
<DocumentFragment>
  <div>
    <div
      class="navbar"
    >
      <i
        class="fab fa-github"
      />
      <span>
        GitHub Search
      </span>
    </div>
    <div
      class="search"
    >
      <form
        class="form"
      >
        <input
          class="input"
          name="github-user"
          placeholder="Github username ..."
          type="text"
          value=""
        />
        <button
          class="button"
        >
          Search
        </button>
      </form>
    </div>
    <div
      class="card-loading"
    >
      <i
        class="fas fa-spinner"
      />
      <span>
        Carregando ....
      </span>
    </div>
    <div
      class="footer"
    >
      <span>
        Created by VSwerts
      </span>
      <div
        class="contact"
      >
        <a
          href="https://github.com/VanessaSwerts"
        >
          <i
            class="fab fa-github-square"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/vanessaswerts/"
        >
          <i
            class="fab fa-linkedin"
          />
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
